toolkit/tools/specreader/specreader.go

```diff
@@ -200,8 +244,40 @@ func createChroot(workerTar, buildDir, specsDir, srpmsDir string) (chroot *safec
 	return
 }
 
+func findSpecFiles(specsDir string, parseList map[string]bool) (specFiles []string, err error) {
+	// Find the filepath for each spec in the SPECS directory.
+	if len(parseList) == 0 {
+		specSearch, err := filepath.Abs(filepath.Join(specsDir, "**/*.spec"))
+		if err == nil {
+			specFiles, err = filepath.Glob(specSearch)
+		}
+		if err != nil {
+			logger.Log.Errorf("Failed to find *.spec files. Check that %s is the correct directory. Error: %v", specsDir, err)
+			return nil, err
+		}
+	} else {
+		for specName := range parseList {
+			var specFile []string
+
+			specSearch := filepath.Join(specsDir, fmt.Sprintf("**/%s.spec", specName))
+			specFile, err = filepath.Glob(specSearch)
```

Comment:
Perhaps `matchingSpecFiles` instead of `specFile`? It is strange seeing a variable being used as a list when it has a name that isn't plural.